src/corelibs/U2Core/src/datatype/msa/MultipleChromatogramAlignment.cpp

```diff
@@ -604,7 +604,7 @@ MultipleChromatogramAlignment MultipleChromatogramAlignmentData::getExplicitCopy
 void MultipleChromatogramAlignmentData::copy(const MultipleAlignmentData &other) {
     try {
         copy(dynamic_cast<const MultipleChromatogramAlignmentData &>(other));
-    } catch (std::bad_cast) {
+    } catch (std::bad_cast &) {
```

Comment:
I suppose, it's better to catch `const std::bad_cast &`

Reply:
I also do not like the code here, but behavior changes are out of scope in this PR

Reply:
In this case, why did you change `std::bad_cast` to `std::bad_cast &`?

Reply:
The warning (-Wcatch-value) was not related to the empty catch clause but to the way the value is caught. See other places with non-empty catch clause that were also changed .